chromium_src/chrome/browser/unexpire_flags.cc

```diff
@@ -18,7 +18,8 @@ namespace flags {
 
 bool IsFlagExpired(const flags_ui::FlagsStorage* storage,
                    const char* internal_name) {
-#if BUILDFLAG(ENABLE_BRAVE_VPN) || BUILDFLAG(ENABLE_PLAYLIST)
+#if BUILDFLAG(ENABLE_BRAVE_VPN) || BUILDFLAG(ENABLE_PLAYLIST) || \
+    defined(TOOLKIT_VIEWS)
```

Comment:
this doesn't seem correct, what are we trying to do here? Only enable this feature for certain channels? This doesn't seem like the right way to do that

Reply:
Yes, it's for controlling channel. I've tried https://github.com/brave/brave-core/pull/14198/files#r922671148 but was reverted due to CI failure. 

https://github.com/brave/brave-core/pull/14291

Reply:
Maybe we could have avoided CI failure of `AllowForDevVersion() - link  above` with `base::check_is_test`. What do you think about the `AllowForDevVersion()`? If you prefer this way, I can try it again.